tubular/scripts/dd_synthetic_tests.py

```diff
@@ -0,0 +1,313 @@
+#! /usr/bin/env python3
+
+import click
+import logging
+import os
+import requests
+import time
+import sys
+
+class SyntheticTest:
+    '''
+    Attributes for a Datadog synthetic test and its test run
+    '''
+    def __init__(self, name, public_id):
+        self.name = name            # The test's Datadog name
+        self.public_id = public_id  # The test's Datadog Test ID
+        self.test_run_id = None     # The run ID given by Datadog to this test's invocation
+        self.success = None
+
+class DatadogClient:
+    ''' Client class to invoke datadog API to run and monitor synthetic tests '''
+
+    DATADOG_SYNTHETIC_TESTS_API_URL = "https://api.datadoghq.com/api/v1/synthetics/tests"
+    MAX_ALLOWABLE_TIME_SECS = 600 # 10 minutes
+
+    WAFFLE_SWITCH_TEST = SyntheticTest(
```

Comment:
This is a bit of a nitpick, but since 'waffle' is the name of the specific library we use in Django to handle toggles, this name is a bit confusing since it doesn't test 'waffle' functionality at all. Maybe something like `TESTS_ENABLED`? (I am not sure I'm any better at naming 😅 .)

Reply:
I'll need to handle passing configuration settings via the CLI invocation in the next pass. Will include timeouts in that. Good call, thanks.

Reply:
I'll rename to DEPLOYMENT_TESTING_ENABLE_DISABLE_SWITCH

Reply:
Generally the format we'd use is something like `DEPLOYMENT_TESTING_ENABLED`. (Sometimes you also see `ENABLE_DEPLOYMENT_TESTING` but then settings don't sort very well when you're looking at an alphabetized listing!)

Reply:
I've gone with a compromise, as this macro isn't for the boolean itself, but rather for the source of the boolean